projects/objects/freight/protos/IntermodalOfficeContainer.proto

```diff
@@ -0,0 +1,120 @@
+#VRML_SIM R2019a utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
+# An intermodal container is a large standardized shipping container, designed and built for intermodal freight transport, meaning these containers can be used across different modes of transport – from ship to rail to truck – without unloading and reloading their cargo.
+# This container contains an office.
+
+PROTO IntermodalOfficeContainer [
```

Comment:
I am wondering if this is the best name, as google image provides better results for simply "office container" rather than "intermodal office container", so I believe we could simplify the name to just `OfficeContainer`.